app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+	def full_title page_title = ""
+		base_title = "Ruby on Rails Tutorial Sample App"
+		page_title.empty? ? base_title : "#{page_title} | #{base_title}"
+	end
```

Comment:
Indent only 2 spaces
```suggestion
  def full_title page_title
    base_title = "Ruby on Rails Tutorial Sample App"
    page_title.blank? ? base_title : "#{page_title} | #{base_title}"
  end
```

Reply:
done